spaceflights/{{ cookiecutter.repo_name }}/src/requirements.txt

```diff
@@ -1,15 +1,15 @@
-black==v19.10b0
+black==21.5b1
 flake8>=3.7.9, <4.0
 ipython~=7.0
-isort>=4.3.21, <5.0
+isort>=4.3.21, <5.9
 jupyter~=1.0
 jupyter_client>=5.1, <7.0
-jupyterlab==0.31.1
+jupyterlab>=0.31.1, <3.1
 kedro[pandas.CSVDataSet,pandas.ExcelDataSet]=={{ cookiecutter.kedro_version }}
 kedro-viz~=3.1
-nbstripout==0.3.3
-pytest-cov~=2.5
-pytest-mock>=1.7.1, <2.0
-pytest~=5.0
-wheel==0.32.2
-scikit-learn==0.23.2
+nbstripout>=0.3.3, <0.5
+pytest-cov>=2.5, <2.13
+pytest-mock>=1.7.1, <4
+pytest>=6.1.2, <6.3
+scikit-learn>=0.23.2, <0.25
```

Comment:
I think we might want this to be `>=0.24` to get rid of annoying deprecation warnings when you run the pipeline.

Reply:
I optimized for wide boundaries, but personally I don't think it'll matter much in this case. Done!